_includes/event-card.html

```diff
@@ -0,0 +1,30 @@
+<li>
+  <article class="event-card">
+    <svg
+    focusable="false"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
+    x="0px"
+    y="0px"
+    viewBox="0 0 24 24"
+    class="event-card__icon"
+    style="enable-background:new 0 0 24 24;"
+    xml:space="preserve"
+    >
+      <path d="M20,3h-1V1h-2v2H7V1H5v2H4C2.9,3,2,3.9,2,5v16c0,1.1,0.9,2,2,2h16c1.1,0,2-0.9,2-2V5C22,3.9,21.1,3,20,3z M20,21H4V8h16V21z"/>
+      <text transform="matrix(1 0 0 1 5.3281 19.1641)" class="st0 st1">{{ event.date | date: "%d" }}</text>
+    </svg>
+    <div
+    class="event-card__info-column l-stack -vertical"
+    style="--stack-spacing: 0.25rem"
+    >
+    <time
+      class="aside event-card__date"
+      datetime="{{ event.date }}"
+    >{{ event.date | localize: site.lang }}</time>
+    {% if include.limit %}<h3{% else %}<h2{% endif %} class="event-card__title ">
+      <a href="{{ event.url }}" class="event-card__link">{{ event.title }}</a>
+    {% if include.limit %}</h3>{% else %}</h2>{% endif %}
```

Comment:
I guess this was like that before, but here's my 2cents:

I find it really difficult to read this mix of code and html, and prefer something like

```
{% if include.limit %}
  <h3 class="event-card__title">
    <a href="{{ event.url }}" class="event-card__link">{{ event.title }}</a>
  </h3>
{% else %}
  <h2 class="event-card__title">
    <a href="{{ event.url }}" class="event-card__link">{{ event.title }}</a>
  </h2>
{% endif %}
```